soc/arm/st_stm32/stm32h7/soc_m7.c

```diff
@@ -85,6 +85,15 @@ static int stm32h7_init(struct device *arg)
 	/* At reset, system core clock is set to 64 MHz from HSI */
 	SystemCoreClock = 64000000;
 
+	/* Power Configuration */
+#if SMPS
```

Comment:
The compile result is shown below.
soc_m7.c: 89: 9: error: #if with no expression #if SMPS